### DISCUSSION TITLE
Image Regulation On Directus

### DISCUSSION THREAD

**Discussion by JAJAKIM22** [directus/directus#21568] [Feature Requests]
### Summary

Way to  regulate images uploaded to Directus by a user in terms of image format, if image is not for instance a png an error would be shown, if image does not meet specific dimensions of both width and height it would also throw an error

### Motivation

This feature will help in image regulation, only image dimensions specified will be accepted. This will help in terms of a user using directus as a backend wont have to keep on editing images to post on their web apps

### Detailed Design

A directus extension to regulate images uploaded to directus before they are uploaded to the files directory.

### Must Have:
- Error handling
- Image accessibility

### Should Have:
-  Error handling

### Could Have:
- Video validation

### Won't Have:
- Video Validation

### Drawbacks

It may be a breaking change

### Alternatives

Addition of assets url that can be configured to make images available to web application with specific dimensions

### Adoption Strategy

Its a breaking change, it can affect the image file uploader

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
